crates/cli/src/subcommands/build.rs

```diff
@@ -19,7 +19,6 @@ pub fn cli() -> clap::Command {
                 .long("skip_clippy")
                 .short('S')
                 .action(SetTrue)
-                .env("SPACETIME_SKIP_CLIPPY")
```

Comment:
I did this because we've generally backed off of supporting env vars for now - should I keep it in? (The smoketests use it)

Reply:
I did not know this even existed and I think now that we're solving the clippy problem this is irrelevent. If we remove this logic here and somewhere else in our infrastructure we're setting this environment variable then that should have no impact because we're just ignoring it here. I would recommend we remove it as you've done here.